tests/test_models/test_nn/deepar_native/test_loss.py

```diff
@@ -0,0 +1,149 @@
+import pytest
+import torch
+from pytorch_lightning import seed_everything
+
+from etna.models.nn.deepar_native.loss import GaussianLoss
+from etna.models.nn.deepar_native.loss import NegativeBinomialLoss
+
+
+@pytest.mark.parametrize(
+    "loss,loc,scale,weights,expected_mean",
+    [
+        (
+            GaussianLoss(),
+            torch.tensor([[[0.0]]]),
+            torch.tensor([[[1.0]]]),
+            torch.tensor([[[1.0]]]),
+            torch.tensor([[[0.0]]]),
+        ),
+        (
+            GaussianLoss(),
+            torch.tensor([[[10.0]]]),
+            torch.tensor([[[100.0]]]),
+            torch.tensor([[[1.0]]]),
+            torch.tensor([[[10.0]]]),
+        ),
+        (
+            NegativeBinomialLoss(),
+            torch.tensor([[[2.0]]]),
+            torch.tensor([[[0.1]]]),
+            torch.tensor([[[1.0]]]),
+            torch.tensor([[[2.0]]]),
+        ),
+        (
+            NegativeBinomialLoss(),
+            torch.tensor([[[0.6]]]),
+            torch.tensor([[[0.1]]]),
+            torch.tensor([[[1.0]]]),
+            torch.tensor([[[0.6]]]),
+        ),
+    ],
+)
+def test_sample_mean(loss, loc, scale, weights, expected_mean):
+    mean = loss.sample(loc=loc, scale=scale, weights=weights, theoretical_mean=True)
+    torch.testing.assert_close(mean, expected_mean)
+
+
+@pytest.mark.parametrize(
+    "loss,loc,scale,weights",
+    [
+        (GaussianLoss(), torch.tensor([[[0.0]]]), torch.tensor([[[1.0]]]), torch.tensor([[[1.0]]])),
+        (GaussianLoss(), torch.tensor([[[1.0]]]), torch.tensor([[[2.0]]]), torch.tensor([[[1.0]]])),
+        (NegativeBinomialLoss(), torch.tensor([[[2.0]]]), torch.tensor([[[2.0]]]), torch.tensor([[[1.0]]])),
+        (NegativeBinomialLoss(), torch.tensor([[[10.0]]]), torch.tensor([[[0.2]]]), torch.tensor([[[1.0]]])),
+    ],
+)
+def test_sample_random(loss, loc, scale, weights, n_samples=200):
+    seed_everything(0)
+    samples = torch.tensor([])
+    for i in range(n_samples):
```

Comment:
Better to implement it with list comprehension

```
samples = torch.concat([loss.sample(loc=loc, scale=scale, weights=weights, theoretical_mean=False) for _ in range(n_samples)], dim=0)
```

It's antipattern to concat sth in cycle